style_editor.html

```diff
@@ -1,4 +1,12 @@
+<!DOCTYPE html>
+
+<html lang="en">
 <head>
+ <title>Style Editor</title>
+  <meta name="description" content="ProffieOS Style Editor">
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
```

Comment:
This seems redundant, why do we need it?

Reply:
Edited. New version has these lines:
<meta http-equiv="X-UA-Compatible" content="IE=edge">
This should ensure it's not rendered as an older version of Internet Explorer, which tends to be buggy.

  <meta name="description" content="ProffieOS Style Editor: Customize your Proffieboard saber blade animations with this browser based tool.">
This should show up in search engine results.

Reply:
Edited. New version has these lines:
<meta http-equiv="X-UA-Compatible" content="IE=edge">
This should ensure it's not rendered as an older version of Internet Explorer, which tends to be buggy.

  <meta name="description" content="ProffieOS Style Editor: Customize your Proffieboard saber blade animations with this browser based tool.">
This should show up in search engine results.

Reply:
These comments should be in the code, not in the code review.

The X-UA-Compatible thing seems suspicious, I don't think we want or need that. (Unless you have a good reason for it.)

Reply:
It's an older code sir, but it checks out. From what I can gather, it seems to be good practice, although a rather dated one at this point.
If it's been fine without it up till now, it's likely fine to 86 it.